src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -409,27 +429,27 @@ public SlackNotifier(CommitInfoChoice commitInfoChoice) {
     public SlackNotifier(final String baseUrl, final String teamDomain, final String authToken, final boolean botUser, final String room, final String tokenCredentialId,
                          final String sendAs, final boolean startNotification, final boolean notifyAborted, final boolean notifyFailure,
                          final boolean notifyNotBuilt, final boolean notifySuccess, final boolean notifyUnstable, final boolean notifyRegression, final boolean notifyBackToNormal,
-                         final boolean notifyRepeatedFailure, final boolean includeTestSummary, final boolean includeFailedTests,
+                         final boolean notifyRepeatedFailure, final boolean includeTestSummary, final boolean includeFailedTests, final boolean uploadFiles, final String artifactIncludes,
```

Comment:
the only constructor that needs updating is the builder, the rest are deprecated

Reply:
All public constructors are deprecated. Do I need to remove new fields from all of them?